.github/workflows/test.yml

```diff
@@ -0,0 +1,70 @@
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Tests
+
+on:  # https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+  push:
+    branches-ignore:  # build all branches except:
```

Comment:
I think we don't need ignore dependabot. It is possible that dep version change can break the code.

Reply:
this does not ignore dependabot. the problem is that when dependabot creates a PR the build is triggered twice. this config is the workaround for this and ensures it is only executed once. Anyways, I saw just saw you created a renovate config file in the repo. are you planning to use renovate instead of dependabot?

Reply:
yes